src/generic/RelSeries.jl

```diff
@@ -1010,12 +1010,103 @@ function Base.sqrt(a::AbstractAlgebra.RelSeriesElem)
    return asqrt
 end
 
+###############################################################################
+#
+#  Derivative and Integral
+#
+###############################################################################
+
+@doc Markdown.doc"""
+    derivative(f::RelSeriesElem{T}) -> RelSeriesElem
+
+Return the derivative of the power series $f$.
+
+julia> R, x = PowerSeriesRing(QQ, 10, "x")
+(Univariate power series ring in x over Rationals, x + O(x^11))
+
+julia> f = 2 + x + 3x^3
+2 + x + 3*x^3 + O(x^10)
+
+julia> derivative(f)
+1 + 9*x^2 + O(x^9)
```

Comment:
````suggestion
```jldoctes
julia> R, x = PowerSeriesRing(QQ, 10, "x")
(Univariate power series ring in x over Rationals, x + O(x^11))

julia> f = 2 + x + 3x^3
2 + x + 3*x^3 + O(x^10)

julia> derivative(f)
1 + 9*x^2 + O(x^9)
```
````

Reply:
Somehow I cannot apply this suggestion myself, strange.